internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -242,12 +252,22 @@ func (c *ClusterManager) processProvisioningClusters() []error {
 	// process each local pending cluster and compare to the underlying ocm cluster
 	for i := range provisioningClusters {
 		provisioningCluster := provisioningClusters[i]
-		glog.V(10).Infof("provisioning cluster ClusterID = %s", provisioningCluster.ClusterID)
-		metrics.UpdateClusterStatusSinceCreatedMetric(provisioningCluster, api.ClusterProvisioning)
-		_, err := c.reconcileClusterStatus(&provisioningCluster)
-		if err != nil {
-			errs = append(errs, errors.Wrapf(err, "failed to reconcile cluster %s status", provisioningCluster.ClusterID))
-			continue
+		if provisioningCluster.ClusterType != api.Enterprise.String() {
+			glog.V(10).Infof("provisioning cluster ClusterID = %s", provisioningCluster.ClusterID)
+			metrics.UpdateClusterStatusSinceCreatedMetric(provisioningCluster, api.ClusterProvisioning)
+			_, err := c.reconcileClusterStatus(&provisioningCluster)
+			if err != nil {
+				errs = append(errs, errors.Wrapf(err, "failed to reconcile cluster %s status", provisioningCluster.ClusterID))
+				continue
+			}
+		} else {
+			provisioningCluster.Status = api.ClusterStatus(api.ClusterProvisioned.String())
+			updateErr := c.ClusterService.Update(provisioningCluster)
+			if updateErr != nil {
+				errs = append(errs, errors.Wrapf(updateErr, "failed to update status of accepted %s cluster %s", api.Enterprise.String(), provisioningCluster.ID))
+				continue
```

Comment:
Same comment as before: if this `continue` is here just to skip the log, it can be replaced with an `else`